test/test_tensorclass_nofuture.py

```diff
@@ -533,20 +729,29 @@ class MyDataParent:
     X_clone = X.clone()
     td = TensorDict({}, batch_size)
     td_clone = td.clone()
-    data_nest = MyDataNest(X=X, batch_size=batch_size)
-    data = MyDataParent(X=X, y=data_nest, z=td, W=W, batch_size=batch_size)
+    data_nest = MyDataNest(X=X, v="test_nested", batch_size=batch_size)
+    v = "test_tensorclass"
+    data = MyDataParent(X=X, y=data_nest, z=td, W=W, v=v, batch_size=batch_size)
     data_nest_clone = data_nest.clone()
     assert type(data_nest_clone) is type(data_nest)
     data.y = data_nest_clone
-    assert data.tensordict["y"] is not data_nest.tensordict
-    assert data.tensordict["y"] is data_nest_clone.tensordict, (
-        type(data.tensordict["y"]),
+    assert data.y.tensordict is not data_nest.tensordict
```

Comment:
do not understand this change. shouldn't it be simply "data.y is data_nest_clone"?